src/filefinder.cpp

```diff
@@ -68,6 +68,7 @@ namespace {
 	constexpr const auto SOUND_TYPES = Utils::MakeSvArray(
 			".opus", ".oga", ".ogg", ".wav", ".mp3", ".wma");
 	constexpr const auto FONTS_TYPES = Utils::MakeSvArray(".fon", ".fnt", ".bdf", ".ttf", ".ttc", ".otf", ".woff2", ".woff");
+	constexpr const auto TEXT_TYPES = Utils::MakeSvArray(".txt", ".csv");
```

Comment:
Silly request: Could we cover more file formats?
.svg .xml .json .yml... Don't need to add any code to interpret or format those files.

That would help me with some other projects.

Reply:
Absolutely possible. I think .svg, .xml, .json, .yaml are all a good start, I can add those.